tk_toolchain/cmd_line_tools/tk_build_resource/__main__.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2019 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+from tk_toolchain.cmd_line_tools.tk_run_app import main
+import sys
+
+sys.exit(main())
```

Comment:
Why not having the main content in this file instead? 

The only reason IMO to have it in a function is if we would like to call it from a different content. But I don't think we want to do that...